antarest/study/web/studies_blueprint.py

```diff
@@ -79,9 +79,10 @@ def get_comments(
 
     @bp.put(
         "/studies/{uuid}/comments",
-        status_code=HTTPStatus.NO_CONTENT.value,
+        status_code=HTTPStatus.NO_CONTENT,
         tags=[APITag.study_raw_data],
         summary="Update comments",
+        response_model=None,
```

Comment:
If you have no particular reason to set `response_model=None` it can be removed instead

Reply:
En fait, les versions récentes de FastAPI peuvent se baser sur la signature de la fonction au lieu de `response_model`. Je peux voir ça dans ma future PR.